modules/tinymce/src/core/main/ts/api/dom/StyleSheetLoader.ts

```diff
@@ -180,71 +164,51 @@ export function StyleSheetLoader(documentOrShadowRoot: Document | ShadowRoot, se
 
     // Start loading
     state.status = 1;
-    // TODO: Use Sugar to create this element
-    link = doc.dom.createElement('link');
-    link.rel = 'stylesheet';
-    link.type = 'text/css';
-    link.id = 'u' + (idCount++);
-    link.async = false;
```

Comment:
Neither of these (async and defer) are valid properties for a link element. This wasn't picked up previously as the type was `any`.